.gitignore

```diff
@@ -7,3 +7,8 @@ relative-json-pointer.txt
 
 # For the Python enviornment
 .venv
+
+package-lock.json
```

Comment:
That shouldn't be there. I'll remove it. My philosophy is that package-lock should be checked-in for applications, but not for libraries. Most of the code I work on is for libraries, so that got added by mistake due to force of habit.